storops/unity/resource/lun.py

```diff
@@ -18,14 +18,163 @@
 import logging
 
 from storops.unity.resource import UnityResource, UnityResourceList
+import storops.unity.resource.pool
+from storops.unity.enums import TieringPolicyEnum, NodeEnum, HostLUNAccessEnum
+from storops.unity.resource.storage_resource import UnityStorageResource
+from storops.exception import UnityResourceNotFoundError
 
 __author__ = 'Jay Xu'
 
 log = logging.getLogger(__name__)
 
 
 class UnityLun(UnityResource):
-    pass
+    @classmethod
+    def create(cls, cli, name, pool, size, sp=None, host_access=None,
+               is_thin=None, description=None, iolimit_policy=None,
+               is_repl_dst=None, tiering_policy=None, snap_schedule=None):
+
+        pool_clz = storops.unity.resource.pool.UnityPool
+        pool = pool_clz.get(cli, pool)
+        TieringPolicyEnum.verify(tiering_policy)
+
+        if sp is None:
+            spNode = NodeEnum.SPA
+        else:

```

Comment:
Better to conclude it in:

```python
if isinstance(sp, UnityStorageProcessor):
    spNode = sp.to_node_enum()
else:
    NodeEnum.verify(spNode)
```

And normally we use `sp_node` instead of `spNode` for variable name.